### PR TITLE
Reorganize cluster logging per F2F

### PR DESCRIPTION
**Pull request by mburke5678**
Multiple discussions around reorganizing the cluster logging docs arose at the cluster logging F2F in March 2020. 
Make the docs less EFK-centric
Better flow needed
Isolate or remove procedures that require unmanaged state as unsupported
Etc.

Version to be determined if/when we decide to proceed. Labeling as 4.5 for now.

Christian review doc: https://docs.google.com/document/d/1Ca04KEUW8-J1z_LuesiD_QivDSFMGTAqiJuft2aI_jw/edit?usp=sharing

### CODE REVIEW

**Comment by mburke5678** [comment]
Version to be determined if/when we decide to proceed. Labeling as 4.5 for now.

**Comment by mburke5678** [comment]
@sheriff-rh @aburdenthehand FYI. Don't let me forget to consider the Dedicated and CNV mark ups in these docs.

**Comment by vikram-redhat** [comment]
@mburke5678 can you work with Stephanie or Ali to confirm that they are also ok with the reorg. Specifically, I am worried about the sub-directory names.

**Comment by sjstout** [comment]
The reorg looks good to me. Thanks, Michael!

**Comment by aburdenthehand** [comment]
@mburke5678 
We changed the prefix for all our files from 'cnv-' to 'virt-' yesterday. 
The file `cluster-logging-about-components.adoc` is the only impacted module in this PR
This applies to the ifdef statement, the related conditionals in the module, and the metadata should be updated to reflect that it is also included in  `virt/logging_events_monitoring/virt-openshift-cluster-monitoring.adoc'
Thank you!

**Comment by aburdenthehand** [comment]
@mburke5678 Looks good to me.
Just for completeness, can you please also add to the `cluster-logging-about-components.adoc` module metada that it is included in the `virt/logging_events_monitoring/virt-openshift-cluster-monitoring.adoc' assembly? Then it will be 100%
Thank you

File: logging/config/cluster-logging-unmanaged.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+:context: cluster-logging-unmanaged
+[id="cluster-logging-unmanagedr"]
+= Unsupported configurations
+include::modules/common-attributes.adoc[]
+
+
+
+
+
+The supported way of configuring {product-title} cluster logging is by configuring it using the options described in this documentumentation. *Do not use other configurations, as they are unsupported.* Configuration paradigms might change across {product-title} releases, and such cases can only be handled gracefully if all configuration possibilities are controlled. If you use configurations other than those described in this documentation, your changes will disappear because the cluster-monitoring-operator reconciles any differences. The operator reverses everything to the defined state by default and by design.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply logging/config/cluster-logging-unmanaged.adoc:10]: Sorry. Cut and paste error. Thanks for catching this.

File: logging/config/cluster-logging-unmanaged.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+You must set the operator to the _unmanaged_ state in order to modify the components managed by the Cluster Logging Operator or the Elasticsearch Operator. An unmanaged cluster logging environment is not supported and does not receive updates until you return the Cluster Logging Operator to Managed state.
+
+Explicitly unsupported cases include:
+
+* *Configuring the collected log location*. You must set the cluster to `Unamanged` to change the location of the log collector output file, by default `/var/log/fluentd/fluentd.log`.
+
+* *Throttling log collection*. You must set the cluster to `Unamanged` to throttle down the rate at which the logs are read in by the log collector
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply logging/config/cluster-logging-unmanaged.adoc:18]: Correct. I also removed the Curator bullets below this, as the topics have been removed (as part of index management).

**Comment by mburke5678** [comment]
Closing in favor of: https://github.com/openshift/openshift-docs/pull/23502

**Pull request closed without merging**